week5_R_draft.ipynb

```diff
@@ -1,32 +1,8 @@
 {
```

Comment:
<p>This is more on aesthetics but I think It'd be clearer if you divide stuff more (like more space between examples and datatables) to have things more spread out and discrete. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-MzCorona/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Got it! Will do!</p>